File: src/main/java/com/team1/qrnavigationproject/model/OrganizationTheme.java
package com.team1.qrnavigationproject.model;


import lombok.Getter;

@Getter
public class OrganizationTheme {
    final String name;
    final String logoURL;
    final String headerBackground;
    final String footerBackground;
    final String websiteURL;

    public OrganizationTheme(Organization organization) {
        this.name = organization.getName();
        this.logoURL = organization.getLogoURL();
        this.headerBackground = organization.getHeaderBackground();
        this.footerBackground = organization.getFooterBackground();
        this.websiteURL = organization.getWebsiteURL();
    }

}

File: app/src/main/java/com/daveace/qrnavigationapp/data/OrganizationTheme.kt
package com.daveace.qrnavigationapp.data

data class OrganizationTheme(val organization: Organization) {
    val name = organization.name
    val logoURL = organization.logoURL
    val headerBackground = organization.headerBackground
    val footerBackground = organization.footerBackground
    val websiteURL = organization.websiteURL
}